### PR TITLE
PSR-4 composer + autowire and register commands (Symfony 3.3+)

### PR DESCRIPTION
**Pull request by angelsk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,13 +15,12 @@
...
-        "psr-0": {
-            "Mopa\\Bundle\\BootstrapBundle": ""
+        "psr-4": {
+            "Mopa\\Bundle\\BootstrapBundle\\": ""
         }
     },
-    "target-dir": "Mopa/Bundle/BootstrapBundle",
     "require": {
-        "symfony/symfony": "~2.3|~3.0",
+        "symfony/symfony": "^3.3",
```
Comment (gavinlove) [inline composer.json:23]: Is this the right limiter? or will this not work with 4 or 5?

Comment (angelsk) [reply composer.json:23]: I don't think the bundle will work with anything greater than 3 atm.  This is just a fix to get it working properly on mybuilder/mybuilder without throwing deprecations.

Comment (angelsk) [reply composer.json:23]: I wouldn't want to commit to saying it'll work with 4, as I don't really know how it works

Comment (angelsk) [reply composer.json:23]: The repo it's branched from works with Symfony 4 I think.  So might be a simple task.

Comment (gavinlove) [reply composer.json:23]: sure, makes sense

**Review by iamdto — APPROVED** [review]

**Pull request merged**